src/dbos.ts

```diff
@@ -897,7 +897,7 @@ export class DBOS {
    * @param input - `GetWorkflowsInput` predicate for filtering returned workflows
    * @returns `GetWorkflowsOutput` listing the workflow IDs of matching workflows
    */
-  static async getWorkflows(input: GetWorkflowsInput): Promise<GetWorkflowsOutput> {
+  static async getWorkflows(input: GetWorkflowsInput): Promise<WorkflowStatus[]> {
```

Comment:
The public API should be identical to Python too -- we should rename it to `DBOS.listWorkflows`, and also change it in the below code:
```js
return await DBOS.#runAsWorkflowStep(async () => {
      return await DBOS.executor.getWorkflows(input);
    }, 'DBOS.listWorkflows');
```

Reply:
If we add `listWorkflows` with the new API, we could leave `getWorkflows` with the original API w/o a breaking change